.gitmodules

```diff
@@ -13,6 +13,7 @@
 [submodule "third/imgui"]
 	path = third/imgui
 	url = https://github.com/ocornut/imgui.git
+	branch = docking
```

Comment:
https://git-scm.com/docs/gitmodules

Didn't know this was possible :ok_hand: .